nec_tables.py

```diff
@@ -438,4 +438,200 @@
                  5000: {'Cu': '700', 'Al': '1200'},
                  6000: {'Cu': '800', 'Al': '1200'}}
 
-conduit_type = []
+# Percent of Cross Section of conduit for Conductors and Cables from NEC 2017 Chapter 9, Table 1
+# (XX,YY)  :: XX=conductor/cable count, YY=conduit_xsection max fill
+condit_xsection =  [(1,0.53),
+                    (2,0.31),
+                    (3,0.40)]
+
+conduit_type = ['EMT',
+                'ENT',
+                'FMC',
+                'IMC',
+                'LFNC-A',
+                'LFNC-B',
+                'LFNC-C',
+                'LFMC',
+                'RMC',
+                'PVC80'.
```

Comment:
Typo here `.` (period) instead of `,` (comma)

Reply:
Corrected.